requirements.txt

```diff
@@ -38,6 +38,7 @@ coverage==7.2.5
 croniter==1.3.8
 cryptography==39.0.2
 dateparser==1.1.7
+dbt-automation @ git+https://github.com/DalgoT4D/dbt-automation.git@33bf2d8778b2995bc6809496e29c1efb7ee0b345
 dbt-core==1.4.5
```

Comment:
I think this should be `dbt-automation 0.1`, thats what it shows in my local django setup

Reply:
throws me an error when i do this
```ERROR: Could not find a version that satisfies the requirement dbt-automation==0.1 (from versions: none)
ERROR: No matching distribution found for dbt-automation==0.1
```